docs/about/products.md

```diff
@@ -0,0 +1,72 @@
+
+**Mono Runtime** - The runtime implements the ECMA Common Language Infrastructure (CLI). The runtime provides a Just-in-Time (JIT) compiler, an Ahead-of-Time compiler (AOT), a library loader, the garbage collector, a threading system and interoperability functionality.
+
+**Base Class Library** - The Mono platform provides a comprehensive set of classes that provide a solid foundation to build applications on. These classes are compatible with Microsoft’s .Net Framework classes.
+
+**Mono Class Library** - Mono also provides many classes that go above and beyond the Base Class Library provided by the .NET Framework. These provide additional functionality that are useful, especially in building Linux applications. Some examples are classes for Gtk+, Zip files, LDAP, OpenGL, Cairo, POSIX, etc.
+
+Unity
+-----
+
+[Unity](http://unity3d.com) is a [popular](http://unity3d.com/public-relations) cross-platform 2D + 3D game engine that uses C# for scripting game play and logic. Unity has several graphics API back-ends including DirectX for Windows and OpenGL for any Operating System (OS) that supports it. A major benefit of Unity is its broad platform support. You can build a single game and publish it to the more than [25 platforms that Unity supports](http://unity3d.com/unity/multiplatform). These platforms include all the popular targets in desktop, console, mobile, VR and AR gaming.
+
+The [Made with Unity](http://madewith.unity.com) site showcases published Unity games. These games demonstrate the breadth of possibility using Unity.
+
+Unity has traditionally used the Mono runtime to execute "C# scripts". More recently, Unity has been adopting its [IL2CPP](http://blogs.unity3d.com/2014/05/20/the-future-of-scripting-in-unity/) technology.
+
+.NET Native
+-----------
+
+.NET Native is a set of native build tools for .NET Core. .NET Native is an Ahead-of-Time (AOT) toolchain that produces native applications by compiling IL byte code to native machine code, . This means that the resulting binary is what the OS executes; there is no JIT-ing, no runtime compilation. This leads to better startup performance, as well as some security benefits.
+
+.NET Native is primarily used by .NET [Universal Windows Platform (UWP)](https://msdn.microsoft.com/library/windows/apps/dn726767.aspx) applications.
+
+Windows 10 IoT Core
+-------------------
+
+[Windows 10 IoT Core](https://developer.microsoft.com/windows/iot) is a version of Windows 10 that is optimized for smaller devices with or without a display, and that runs on the Raspberry Pi 2 and 3, Arrow DragonBoard 410c & MinnowBoard MAX. Windows 10 IoT Core utilizes the rich, extensible Universal Windows Platform (UWP) API for building great solutions.
+
+You can build IoT apps in C# using .NET APIs using Visual Studio, with rich debugging of your app running on-device. You can see [examples of IoT apps](https://www.hackster.io/windowsiot) that have been built.
```

Comment:
Note that you can also use F# (and maybe VB?), but it has to be a PCL that is called from C# code which uses the SDK.